src/Impl/GuzzleHttpRequester.php

```diff
@@ -41,18 +40,15 @@ public function executePost(
     }
```

Comment:
For usage in publishMulti method: this method now returns array of success/failure objects.

For backwards compatibility, the existing single-message publish methods, still analyze if the single response was a failure and throw exception in such case, or return success.

Not the case for multimessage processing - we need to collect all results, and keep on processing even on error caused by a message in the middle of the batch